### ISSUE TITLE
Spritefont Build Errors

### ISSUE THREAD

**Issue by EddyLuten** [proc-gen/roguelite-survivor#80]
Hi, I ran into a build error on Ubuntu 22.04.2 LTS when compiling the game after pulling it from the repo:

```
The command "dotnet mgcb /quiet /@:"/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/Content/Content.mgcb" /platform:DesktopGL /outputDir:"/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/Content/bin/DesktopGL/Content" /intermediateDir:"/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/Content/obj/DesktopGL/net7.0/Content" /workingDir:"/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/Content/"" exited with code 2. [/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/RogueliteSurvivor.csproj]
```

Somewhere earlier in the same build log:

```
  /home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/Content/Fonts/Font.spritefont
/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/Content/Fonts/Font.spritefont : error : Processor 'FontDescriptionProcessor' had unexpected failure! [/home/eddy/code/roguelite-survivor/RogueliteSurvivor/RogueliteSurvivor/RogueliteSurvivor.csproj]
```

I worked around this by changing the `<FontName>` entries in the `.spritefont` files from Calibri to another font available on my system since Calibri isn't available on non-Windows platforms.

**Comment by proc-gen** [comment]
Thanks for the heads up! 

This one will be a little more time consuming to fix because I'll also need to remake a bunch of graphics to match whatever new font gets chosen. It'll also give me an excuse to finally name the files properly instead of `Font` and `FontSmall`.

**Comment by proc-gen** [comment]
I updated the Spritefonts to both use Arial for the time being. It did cause a little bit of a spacing issue for the HUD text as it does take more screen space, so for now I changed `Spell Effect Chance` to `Effect Chance` and `Area of Effect` to `AOE`.

I didn't notice any major glaring issues by having Arial in game and Calibri for all the textures I'd created, so that's a little bit lower priority on getting updated.
